src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -231,4 +248,60 @@ void parse_processor_info_linux(const int _processors,
     }
 };
 
+void get_cpu_mapping_from_cores(const int _processors,
+                                const int _sockets,
+                                const int _cores,
+                                const int _big_phys_cores,
+                                std::vector<std::vector<int>>& _proc_type_table,
+                                std::vector<std::vector<int>>& _cpu_mapping_table) {
+    const auto hyper_thread = _processors > _cores ? true : false;
+    const auto num_big_cores = hyper_thread ? _big_phys_cores * 2 : _big_phys_cores;
```

Comment:
When TBB cannot report right number of big core, we can get right number of big core by _processors - _cores.

Reply:
Ok, done.